### PR TITLE
Pytorch based network

### PR DESCRIPTION
**Pull request by jedrzejpolaczek**

### CODE REVIEW

File: predict_utils/load_checkpoint.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        torchvision.models: object of the proper architecture class.
+    """
+    if arch == 'resnet18':
+        model = torchvision.models.resnet18(pretrained=True)
+    if arch == 'alexnet':
+        model = torchvision.models.alexnet(pretrained=True)
+    if arch == 'vgg13':
+        model = torchvision.models.vgg13(pretrained=True)
+    if arch == 'vgg16':
+        model = torchvision.models.vgg16(pretrained=True)
```
Comment (azjawinski) [inline predict_utils/load_checkpoint.py:23]: What in the case of some spelling error? 🙈 
Maybe we should add a `throw` here for explicit flow interrupt rather than hoping nothing will break later on?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...

+# To regenerate from the project's Pipfile, run:

+#

+#    pipenv lock --requirements

+#

+
```
Comment (azjawinski) [inline requirements.txt:7]: Is this file needed?

File: train_utils/get_arch.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        torchvision.models: object of the proper architecture class.
+    """
+    if arch == 'resnet18':
+        model = torchvision.models.resnet18(pretrained=True)
+    if arch == 'alexnet':
+        model = torchvision.models.alexnet(pretrained=True)
+    if arch == 'vgg13':
+        model = torchvision.models.vgg13(pretrained=True)
+    if arch == 'vgg16':    
+        model = torchvision.models.vgg16(pretrained=True)
```
Comment (azjawinski) [inline train_utils/get_arch.py:22]: Why the same code is duplicated in `get_arch` and `load_checkpoint`?

File: train_utils/get_loader.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+
+def get_transforms() -> dict:
+    """
+    Create functions for transforming training, validation and test data.
+
+    Return:
+        dict: dictionary of transformation function for train, validation and test data.
+    """
+    train_transform = transforms.Compose([transforms.RandomRotation(30),
```
Comment (azjawinski) [inline train_utils/get_loader.py:31]: 30 degrees is a lot, we need to be aware of the black areas that probably take a significant amount of the space after the rotation 🤔 

Though @WlodarczykKacper said the model started working for him, maybe it's not such a big issue 🤔

**Review by azjawinski — APPROVED** [review]
Looks good 🎉 
Had only a few non-urgent comments.

File: train_utils/get_arch.py (hunk 3 above)
Comment (jedrzejpolaczek) [reply train_utils/get_arch.py:22]: It is due to a little problem to organise that part of the code.

File: train_utils/get_loader.py (hunk 4 above)
Comment (jedrzejpolaczek) [reply train_utils/get_loader.py:31]: Yeah, but hey that's the part of the experiment!

Comment (WlodarczykKacper) [reply train_utils/get_loader.py:31]: One suggestion from me
It actually makes it harder for network to obtain low loss on training dataset, and by that it's the reason (or at least one of the reasons) why train loss was higher than validation loss.
